manifests/cirunner/docker.pp

```diff
@@ -0,0 +1,70 @@
+# == Class: gitlab::cirunner::docker
+#
+class gitlab::cirunner::docker (
+  Boolean                    $manage_docker                     = true,
+  String                     $xz_package_name                   = 'xz-utils',
+  String                     $default_image                     = 'ubuntu_trusty',
```

Comment:
I think it might be better to make something a bit leaner the default image. Perhaps `docker`? Not going to make this a review comment, just a discussion I want to start.